jumanji/environments/routing/maze/generator.py

```diff
@@ -72,8 +86,35 @@ def __call__(self, key: chex.PRNGKey) -> chex.Array:
             key: the Jax random number generation key.
 
         Returns:
-            maze: A generated maze as an array of booleans.
+            state: the generated state.
         """
-        return maze_generation.generate_maze(self.num_cols, self.num_rows, key).astype(
-            bool
+        key, maze_key, agent_key = jax.random.split(key, 3)
+
+        walls = maze_generation.generate_maze(
+            self.num_cols, self.num_rows, maze_key
+        ).astype(bool)
```

Comment:
Are the first two arguments to this function in the correct order? I ask because `generate_maze` states:

```
def generate_maze(width: int, height: int, key: chex.PRNGKey) -> chex.Array:
    """Randomly generate a maze.

    Args:
        width: the number of rows of the maze to create.
        height: the number of columns of the maze to create.
        key: the Jax random number generation key.
```
so it seems that the first argument should be the number of rows while the second should be the number of columns?

Reply:
Hi @dluo96 - I think the arguments in `maze_generation.generate_maze(....)` are in the correct order. But I think the `generate_maze` doc string is incorrect. The `width` of a maze is the number of columns (not rows) the maze has, and the `height` should be the number of rows (not cols).

Reply:
Good spot though!

Reply:
Thanks @dluo96! 
I believe this now closes Issue https://github.com/instadeepai/jumanji/issues/131